chapters/convertToAbbreviatedForm.typ

```diff
@@ -1,11 +1,10 @@
-#import "@preview/tablex:0.0.4": tablex, colspanx, rowspanx, cellx
+#import "@preview/tablex:0.0.4": tablex, colspanx, rowspanx, cellx, hlinex
 
-= Refactoring — Convert Function Template to Abbreviated Form <convert_to_abbreviated_form>
-For the second refactoring a subset of the third idea (@third_idea) was implemented.
+= Refactoring — Abbreviate Function Template <abbreviate_function_template>
+For the second refactoring, a subset of the third idea (@third_idea) should be implemented.
```

Comment:
```suggestion
For this refactoring, a subset of the ide in @third_idea is implemented.
```

Reply:
"was implemented" oder "has been implemented" findi iwie besser. wa meinsch?

Reply:
ich find eigentlich "is implemented" ganz ok, s seit es isch abgschlosse. https://oneminuteenglish.org/en/is-done-has-been-done/

Aber bin mer ned 100% sicher was besser isch, han s gfühl das es irgendwie komisch isch ganz ide vergangeheit z schriibe. Bi "Has been implemented" tönts so als wärs ewig her

Reply:
Antwort bruce: "du nimmsch praktisch nie vergangeheitsforme i wüsseschaftliche text. drum "is implemented". zudem würdi praktisch nie, this schriebe. du referenziersch uf us wüsseschaftlicher sicht z ungnau mit this"